sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_multiple_node/language-modeling-multi-gpu-multi-node.ipynb

```diff
@@ -2,15 +2,15 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    NUM_INSTANCES = 2
</pre><p>A previous paragraph in this notebook was revised to say that this example uses four instances - however this sets it at two. Do you want to use 2 or 4?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3054/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Since this notebook runs two training jobs, each uses 2 instances, so in total the notebook requires 4.

Reply:
I see, thanks for clarifying.